src/components/Asset/AssetActions/Trade/Swap.tsx

```diff
@@ -144,7 +144,9 @@ export default function Swap({
 
         setBaseTokenItem((prevState) => ({
           ...prevState,
-          amount: amountBaseToken.toDecimalPlaces(MAX_DECIMALS).toString(),
+          amount: new Decimal(balance.baseToken)
```

Comment:
Not sure if correct but the change translates in: I have this amount of base tokens from my wallet, but I ca only spend the maximum amount of token, so the token balance displayed is the one from my wallet and doesn't depend on any other calculations based on pool data.

![Screenshot from 2022-04-04 23-37-42](https://user-images.githubusercontent.com/49017601/161629570-ae07d583-f447-4b70-8e2b-e323b5c149e2.png)